spec/index.bs

```diff
@@ -13720,16 +13720,28 @@ is being composited into (e.g. an HTML page rendering, or a 2D canvas).
         Read RGBA as premultiplied: color values are premultiplied by their alpha value.
         100% red at 50% alpha is `[0.5, 0, 0, 0.5]`.
 
-        If [=out-of-gamut premultiplied RGBA values=] are output to the canvas, and the canvas is:
+        If the canvas texture contains [=out-of-gamut premultiplied RGBA values=] at the time the
+        canvas contents are read, the behavior depends on whether the canvas is:
 
         <dl class=switch>
             : [$get a copy of the image contents of a context|used as an image source$]
             :: Values are preserved, as described in [[#color-space-conversions|color space conversion]].
 
             : displayed to the screen
             :: Compositing results are undefined.
+
+                Note:
                 This is true even if color space conversion would produce in-gamut values before
-                compositing, because the intermediate format for compositing is not specified.
+                compositing, because during compositing implementations may use intermediate
+                representations that can't represent out-of-gamut values. For example, on a
+                {{GPUCanvasAlphaMode/"premultiplied"}}, {{PredefinedColorSpace/"srgb"}},
+                {{GPUTextureFormat/"rgba8unorm"}} canvas, the color `[0.51, 0, 0, 0.5]` represents
+                <code><a funcdef>color</a>(srgb 1.02 0 0 / 0.5)</code>,
+                however upon presentation it could be internally unpremultiplied before color
```

Comment:
It's probably not actually likely that a browser would unpremultiply like this, as basically everything prefers to operate in premultiplied space. I don't think this is the only case under which bad things would happen. It was the easiest to explain, but could it mislead people to believe this is actually a likely scenario?

The example is not critical, but it's very helpful to demonstrate what we're talking about. Maybe it would be possible to show other examples as well of hypothetical compositing+display pipelines?